pom.xml

```diff
@@ -112,16 +112,18 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <!--
+            FIX ME: Have to comment out this plugin since it conflicts with ForkJoinPool dependency.
```

Comment:
I certainly don't mind but let's ask for some other people's opinions.


________________________________
From: IlyaAvdeev <notifications@github.com>
Sent: Wednesday, May 27, 2015 9:15 AM
To: SmartBear/soapui
Cc: Manne Fagerlind
Subject: Re: [soapui] SOAP-3838 new plugin framework (#161)


In pom.xml<https://github.com/SmartBear/soapui/pull/161#discussion_r31108403>:

> @@ -112,16 +112,18 @@
>                      </dependency>
>                  </dependencies>
>              </plugin>
> +            <!--
> +            FIX ME: Have to comment out this plugin since it conflicts with ForkJoinPool dependency.


I'm very happy that you raised up this question. I spent some time while working on plugin framework and was afraid of inadmissibility this decision because of some reasons. If you don't mind of upping Java version to 7 I completely agree!!!

-
Reply to this email directly or view it on GitHub<https://github.com/SmartBear/soapui/pull/161/files#r31108403>.